source/generate.py

```diff
@@ -123,32 +125,33 @@ def generate_multi_track_sequence(
 if __name__ == "__main__":
 
     device = "cpu"
-    # model path
-    # model_path = "models/model_2048_10kseq/"
-    # tokenizer_path = "models/model_2048_10kseq/tokenizer.json"
-    # generated_sequence_files_path = "models/model_2048_10kseq/generated_sequences/"
-
-    # model path
-    model_path = "models/model_2048_wholedataset/"
-    tokenizer_path = "models/model_2048_wholedataset/tokenizer.json"
-    generated_sequence_files_path = (
-        "models/model_2048_wholedataset/generated_sequences/"
-    )
-
+    load_from_huggingface = False
+
+    if load_from_huggingface:
```

Comment:
why don't we make this logic inside of load methods? so something like load_model(path/repo, isLocal=False) because this logic belongs to loading not generating

Reply:
yeahhhhh. makes more sense I will do that.